<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transformination</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::ScrollHandDrag</enum>
        </property>
        <property name="transformationAnchor">
         <enum>QGraphicsView::NoAnchor</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>596</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionPlay_All"/>
    <addaction name="actionReset"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Items</string>
    </property>
    <addaction name="action_Add_Transform"/>
    <addaction name="actionRemove"/>
    <addaction name="actionEdit"/>
    <addaction name="actionApply"/>
    <addaction name="actionReverse_Apply"/>
    <addaction name="actionMove_Up"/>
    <addaction name="actionMove_Down"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionCenter_After_Transform"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="transformListDock">
   <property name="windowTitle">
    <string>TransformList</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListWidget" name="transformListWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlay_All"/>
   <addaction name="actionApply"/>
   <addaction name="actionReverse_Apply"/>
   <addaction name="action_Add_Transform"/>
   <addaction name="actionRemove"/>
   <addaction name="actionMove_Up"/>
   <addaction name="actionMove_Down"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
   <addaction name="actionReset"/>
  </widget>
  <action name="action_Add_Transform">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add Transform</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/document-edit.png</normaloff>:/images/document-edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Edit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionApply">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/apply.png</normaloff>:/images/apply.png</iconset>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
   <property name="toolTip">
    <string>Apply to &quot;after&quot; widget immediately</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/zoom-In-icon.png</normaloff>:/images/zoom-In-icon.png</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/zoom-out-icon.png</normaloff>:/images/zoom-out-icon.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionPlay_All">
   <property name="text">
    <string>Play All</string>
   </property>
  </action>
  <action name="actionMove_Up">
   <property name="text">
    <string>Move Up</string>
   </property>
  </action>
  <action name="actionMove_Down">
   <property name="text">
    <string>Move Down</string>
   </property>
  </action>
  <action name="actionReverse_Apply">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>Reverse Apply</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionCenter_After_Transform">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Center After Transform</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/original.ico</normaloff>:/images/original.ico</iconset>
   </property>
   <property name="text">
    <string>Reset to Original</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
